FlowCrypt/Controllers/Bootstrap/FatalErrorViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  FatalErrorViewController.swift
+//  FlowCrypt
+//
+//  Created by  Ivan Ushakov on 05.12.2021
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import FlowCryptUI
+import AsyncDisplayKit
+
+final class FatalErrorViewController: TableNodeViewController {
```

Comment:
This one is for cases when it is not possible to continue using app and you have to restart it

Reply:
This is very similar to the other error screen we have. It should be the same screen and same VC, with a boolean init parameter to offer to reset storage or not.

Reply:
It is strange to have universal controller with all possible error handling logic.

Reply:
It's not strange to me. A "fatal error" kind of page will always look about the same, no matter the underlying problem. Sometimes it will have a button and sometimes not. But they are very similar, aren't they? I'll file another issue.